app/views/restaurants/_comments.html.erb

```diff
@@ -0,0 +1,29 @@
+<!-- Comment area-->
+<div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well">
+
+    <!-- not in DB -->
+    <h4>Leave a Comment:</h4>
```

Comment:
Tnx. I think I may need to fix sublime.. it auto indents to 4 when i press tab...